components/ai_chat/resources/page/components/input_box/index.tsx

```diff
@@ -119,11 +115,17 @@ function InputBox() {
           <Button
             fab
             kind='plain-faint'
-            onClick={handleSubmit}
+            onClick={() => {
+              if (context.isGenerating) {
+                getPageHandlerInstance().pageHandler.stopGeneration()
+              } else {
+                context.submitInputTextToAPI()
+              }
+            }}
             disabled={context.shouldDisableUserInput}
             title={getLocale('sendChatButtonLabel')}
```

Comment:
If we're reusing the same button, shouldn't we change the title?

Reply:
changed